homeassistant/components/home_connect/sensor.py

```diff
@@ -38,47 +42,206 @@
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterPowderCoffee",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="powder_coffee_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterHotWater",
+        native_unit_of_measurement=UnitOfVolume.MILLILITERS,
+        device_class=SensorDeviceClass.VOLUME,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="hot_water_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterHotWaterCups",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="hot_water_cups_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterHotMilk",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="hot_milk_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterFrothyMilk",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="frothy_milk_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterMilk",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="milk_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterCoffeeAndMilk",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="coffee_and_milk_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="ConsumerProducts.CoffeeMaker.Status.BeverageCounterRistrettoEspresso",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        translation_key="ristretto_espresso_counter",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="BSH.Common.Status.BatteryLevel",
+        device_class=SensorDeviceClass.BATTERY,
+        translation_key="battery_level",
+    ),
+    HomeConnectSensorEntityDescription(
+        key="BSH.Common.Status.BatteryChargingState",
+        device_class=SensorDeviceClass.ENUM,
+        translation_key="battery_charging_state",
+        options_translation_keys={
+            "BSH.Common.EnumType.BatteryChargingState.Charging": "charging",
+            "BSH.Common.EnumType.BatteryChargingState.Discharging": "discharging",
+        },
+    ),
+    HomeConnectSensorEntityDescription(
+        key="BSH.Common.Status.ChargingConnection",
+        device_class=SensorDeviceClass.ENUM,
+        translation_key="charging_connection",
+        options_translation_keys={
+            "BSH.Common.EnumType.ChargingConnection.Connected": "connected",
+            "BSH.Common.EnumType.ChargingConnection.Disconnected": "disconnected",
+        },
+    ),
```

Comment:
Shouldn't we migrate those to binary sensors?

Reply:
Yep, that makes a lot of sense
(I made this binary sensor entities from the first PR I did and I didn't remember that discussion about enum sensors that only have two states, sorry 😅)

Reply:
Are these already in a release?

Reply:
No, I mean this PR that I closed: #125898